reddit_service_websockets/socketserver.py

```diff
@@ -131,6 +134,22 @@ def __init__(self, metrics,
         self.connections = set()
 
     def __call__(self, environ, start_response):
+        try:
+            return self._handle_request(environ, start_response)
+        except (KeyboardInterrupt, SystemExit):
+            raise
+        except:
+            self.error_reporter.http_context({
```

Comment:
do we care that self.error_reporter could technically be `None` here (e.g. in tests below)?